File: task-03/HelloWorld.py
#Python program to print all the prime numbers upto n

def prime(n):
	if(n==1 or n==0):
		return False
	for i in range(2,n):
		if(n%i==0):
			return False

n=int(input("Enter a number- "))
for i in range(1,n+1):
	if(prime(i)!=False):
		print(i)
		

File: task-03/HelloWorld.rb
#Ruby program to print all the prime numbers upto n

def prime(n)
    if n == 0 || n == 1
        return false 
    end
    range = (2..Math.sqrt(n)).to_a
    for i in range 
        if n % i == 0
            return false
        end
    end
    return true
end

puts "Enter a number- "
n = gets.chomp.to_i

range = (1..n)
for i in range 
    if prime(i)
        puts"#{i}"
    end
end
